### ISSUE TITLE
Chromium installation location?

### ISSUE THREAD

**Issue by nucleare2** [ilpersi/BHBot#172]
Is there a way to change the configuration options to use a custom Chromium installation location?

**Comment by ilpersi** [comment]
Yes, you can do this using command-line arguments for the executable.

You can specify

- the chromedriver location using `chromeDriverExePath` parameter
- the chromium folder path using the `chromiumExePath`
- the data directory for chrome profiles (if you plan to run multiple versions) with the `userDataDir `

Here is a sample of me running BHBot with Chrome 83
`BHBot.exe chromeDriverExePath ".\chromedriver83.exe" chromiumExePath "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" userDataDir "./chrome83"`

**Comment by ilpersi** [comment]
@nucleare2 is the answer I provided to you good enough?
This information needs to be added to the documentation.
